packages/shared-base/src/utils/subscription.ts

```diff
@@ -1,5 +1,6 @@
 import { noop } from 'lodash-es'
-import type { Subscription } from 'use-subscription'
+import type { Subscription, Unsubscribe } from 'use-subscription'
+import type { StorageItem, StorageObject } from '..'
```

Comment:
Importing from parent folder like this can easily introduce circular dependencies.